server/src/main/java/com/cloud/resource/ResourceManagerImpl.java

```diff
@@ -909,31 +909,41 @@ protected boolean doDeleteHost(final long hostId, final boolean isForced, final
         // Verify that host exists
         final HostVO host = _hostDao.findById(hostId);
         if (host == null) {
-            throw new InvalidParameterValueException("Host with id " + hostId + " doesn't exist");
+            String errorMessage = String.format("Host with ID [%s] was not found", hostId);
```

Comment:
```suggestion
            String errorMessage = String.format("Host with UUID [%s] was not found", host.getUuid());
```

Reply:
> ```suggestion
>             String errorMessage = String.format("Host with UUID [%s] was not found", host.getUuid());
> ```

This will cause NPE as host is null